opentelemetry-auto-instrumentation/src/opentelemetry/auto_instrumentation/auto_instrumentation.py

```diff
@@ -15,22 +15,33 @@
 # limitations under the License.
 
 from logging import getLogger
-from runpy import run_path
+from os import environ, execl
+from os.path import abspath, dirname, pathsep
+from shutil import which
 from sys import argv
 
-from pkg_resources import iter_entry_points
-
 logger = getLogger(__file__)
 
 
 def run() -> None:
 
-    for entry_point in iter_entry_points("opentelemetry_instrumentor"):
-        try:
-            entry_point.load()().instrument()  # type: ignore
-            logger.debug("Instrumented %s", entry_point.name)
+    python_path = environ.get("PYTHONPATH", [])
```

Comment:
It's just a detail but I think it's better to omit the default parameter.

Reply:
Sorry, better how? :thinking: This is needed to handle the case where the environment variable is not defined.

Reply:
I think your code will work the same if you don't pass anything, the default is None and the conditional you have below could handle it.

Reply:
Ah, I see now, fixing.